src/MicrosoftAi/AbstractionMapper.cs

```diff
@@ -50,9 +50,17 @@ internal static class AbstractionMapper
 	/// <returns>A <see cref="ChatRequest"/> object containing the converted data.</returns>
 	public static ChatRequest ToOllamaSharpChatRequest(IList<ChatMessage> chatMessages, ChatOptions? options, bool stream, JsonSerializerOptions serializerOptions)
 	{
+		object? format = null;
+
+		if (options?.ResponseFormat is ChatResponseFormatJson jsonFormat)
+			if (jsonFormat.Schema is null)
+				format = Application.Json;
+			else
+				format = jsonFormat.Schema;
```

Comment:
```suggestion
		if (options?.ResponseFormat is ChatResponseFormatJson jsonFormat)
		    format = jsonFormat.Schema ?? Application.Json;
```

Reply:
Simplified the logic based on this, but had to take a slightly different approach due to type mismatches. Thanks!